hera_mc/tests/test_librarian.py

```diff
@@ -2,348 +2,358 @@
 # Copyright 2016 the HERA Collaboration
 # Licensed under the 2-clause BSD license.
 
-"""Testing for `hera_mc.librarian`.
-
-"""
+"""Testing for `hera_mc.librarian`."""
 from __future__ import absolute_import, division, print_function
 
-import unittest
 import os
 from math import floor
-import numpy as np
+
+import pytest
 from astropy.time import Time, TimeDelta
 
-from .. import mc, cm_transfer
-from ..librarian import LibStatus, LibRAIDStatus, LibRAIDErrors, LibRemoteStatus, LibFiles
-from .. import utils, geo_location
-from ..tests import TestHERAMC
+from ..librarian import (LibStatus, LibRAIDStatus, LibRAIDErrors,
+                         LibRemoteStatus, LibFiles)
 from hera_mc.data import DATA_PATH
+from .. import utils
+
+time = Time.now() - TimeDelta(6 * 60, format='sec')
```

Comment:
Personally apprehensive about data that needs to be set up but is done globally like this. If your inputs are static over the file you can have a fixture defined for a whole module and have the fixtures pass inputs to functions.

Reply:
yeah, it just seemed harder to put it in a fixture (it used to be in a setup method on a unit test object). I can put it in a fixture.

Reply:
I think it is just how comfortable you are with the global variables. Some people are very against them.

Reply:
fixed.